libs/blocks/article-header/article-header.js

```diff
@@ -33,8 +33,8 @@ function openPopup(e) {
 
 async function buildAuthorInfo(authorEl, bylineContainer) {
   const { textContent } = authorEl;
-  const link = authorEl.href || authorEl.dataset.authorPage;
   const config = getConfig();
+  const link = authorEl.href || authorEl.dataset.authorPage || `${config.locale.contentRoot}/authors/${textContent.replace(/[^0-9a-z]/gi, '-').toLowerCase()}`;
```

Comment:
Nit: you can use `\w` to match any word character or `\W` for non-word characters.
```
.replace(/[\W_]/g, '-')
```

Reply:
I pulled this directly from da-bacom-blog so I feel like I should keep it the same just for the sake of consistency https://github.com/adobecom/da-bacom-blog/blob/3225fb812707b5ae743943d8af581dc6a4f6d565/blog/scripts/scripts.js#L74